app/models/dish.rb

```diff
@@ -35,7 +35,7 @@ def self.get_dish(name)
       raise ArgumentError, "Dish doesn't exist." unless !dish.empty?
       dish.first
     end
-
+ 
```

Comment:
Very important contribution. Without that extra space, the whole project would be doomed.